### ISSUE TITLE
Change POSTGRES env var name to DATABASE_URL

### ISSUE THREAD

**Issue by canadaduane** [meteor-stream/meteor-postgres#238]
I'm a fan of using DB migrations and so my out-of-the-gate focus has been in finding a way to set up my shiny postgres DB in anticipation of using it with meteor-postgres. The following library seems suitable:

https://github.com/theoephraim/node-pg-migrate

However, it uses DATABASE_URL as the environment variable. This seems to be more standard--i.e. the type of database is specified by the uri "protocol", and so it seems unnecessary to call the environment variable name itself "POSTGRES".

In other words, I'm hoping to be able to set one environment variable for both node-pg-migrate and meteor-postgres:

```
DATABASE_URL="postgres://duane@localhost/mydb"
```

**Comment by Eddolan** [comment]
I am :thumbsdown: on this. I wouldn't be against changing POSTGRES to be something more descriptive list PGCONSTRING, but I think that DATABASE_URL is too generic and likely to conflict when developers have more than one DB set up.

**Comment by canadaduane** [comment]
There are two opposing goals we need to satisfy then:


1. Make it possible to configure meteor-postgres independent of other database-using apps, even in the presence of environment variables for the other app(s).




2. Make it possible to share configuration with other apps or services in order to reduce duplicate database configuration.




For 1, may I suggest prefixing the environment variable with MP_ or METEOR_POSTGRES_.




For 2, use the most common config variable in the nodejs ecosystem, I.e. DATABASE_URL.




Then, simply prefer the more specific variable, and fall back on the more generic one:




```

process.env.MP_POSTGRES || process.env.DATABASE_URL

```

**Comment by Eddolan** [comment]
I am :+1: on this.

**Comment by Eddolan** [comment]
Closed via #240. Run meteor update and you should be good to go.
